### DISCUSSION TITLE
Discussion on upcoming 3.30.18 release

### DISCUSSION THREAD

**Discussion by k-doering-NOAA** [nmfs-stock-synthesis/stock-synthesis#429] [Team Posts]
Hi all, just a reminder that we are aiming to get a SS release out by the end of September. Note we have also discussed getting a prerelease out first (to allow users some time to catch bugs that we can fix before the release), so I have modified our checklist slightly: https://github.com/nmfs-stock-synthesis/stock-synthesis/issues/207

Some questions that came up for me:
1. When do we want to get the prerelease out by?
2. How much time do we let the prerelease sit, at a minimum? My thought was about a week
3. Does the [checklists](https://github.com/nmfs-stock-synthesis/stock-synthesis/issues/207#issue-989970496) I put together for before prerelease and before release make sense? Or maybe should we basically do everything as we would before a release during the prerelease, and then we can update the exes to be the official release a week later?
4. Who do we want to test out the prerelease? In theory, we can put it out and not really announce it except perhaps to a group of users we select; another option is we put out the release notes at the time of the prerelease to allow as many users as possible to see it.

**Comment by Rick-Methot-NOAA** [comment]
I think the prerelease should have a short description of the major new features we want people to try out, with enough info on the syntax for invoking the feature.  That will take > a week  to assemble.  I would prefer not to send that to the whole VLAB distribution list and instead send it to a shorter NMFS mainly list plus some key contacts.  I will go through what is in .18 and create the list to be highlighted.

**Comment by k-doering-NOAA** [comment]
OK, sounds good - I don't want the prerelease to create too much extra work for us, so hopefully we can use the info we put together in the manual and in the release notes.

**Comment by Rick-Methot-NOAA** [comment]
What gets written for prerelease can get re-used for the release.  What
gets used for the prerelease should pull from manual where possible.

list is:
all bugs that got fixed
fleet-specific F
predator M2
F at Blimit output

So not much to do to describe these.  Biggest is on me to describe the
predator M2
